packages/qunit-dom/tsconfig.json

```diff
@@ -7,11 +7,12 @@
     "verbatimModuleSyntax": true,
     // We build to ESM, so we need to tell TS that we build to ESM
     // otherwise TS uses fake ESM (you use imports, but it assumes the compilation is to CJS)
-    // These are also required when using 
-    // verbatimModuleSyntax, because it's an 
+    // These are also required when using
+    // verbatimModuleSyntax, because it's an
     // ESM-only feature
     "target": "esnext",
     "module": "esnext",
+    "moduleResolution": "node",
```

Comment:
Let's use `"bundler"` option here instead of `"node"` since it seems like rollup already requires it  in v4 when typescript is involved 

https://vitejs.dev/guide/migration#rollup-4